### PR TITLE
Working Groups dead links

### PR DESCRIPTION
**Pull request by jenniferward**

### CODE REVIEW

**Comment by jenniferward** [comment]
@lpugin @ahankinson What does this mean?

**Comment by ahankinson** [comment]
It just means staging has changed since you started working on this change. Click the 'update branch' button and it should be fine.

File: working-groups.en.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Belarussian Academy of Music\
 Internationulnaja, 30\
 BL-220030 Minsk\
 Email: [shapiro@open.by](mailto:shapiro@open.by)\
-Website: [http://www.bgam.edu.by/eng/](http://www.bgam.edu.by/eng/){:target="_blank"}
+Website: [https://www..by/eng/](https://www.bgam.by/eng/){:target="_blank"}
```
Comment (ahankinson) [inline working-groups.en.md:79]: This is probably not correct...

Comment (jenniferward) [reply working-groups.en.md:79]: Ok. What should I do? I have other things on that page that I need to fix as well.

Comment (ahankinson) [reply working-groups.en.md:79]: You can just fix it... by 'not correct' I mean that you're just missing 'bgam' in the URL... there are two dots.

Comment (jenniferward) [reply working-groups.en.md:79]: Ok, I am about to commit changes for this (thanks for catching) plus some other things on the page.

Comment (ahankinson) [reply working-groups.en.md:79]: 👍

**Pull request merged**